src/app/shared/shared.module.ts

```diff
@@ -21,18 +23,30 @@ const declarations = [components, directives, pipes].reduce(
     []
 );
 
+// services which will be loaded in different way
+const skipServices = ['AuthorizedHttp'];
 // Get only providers, not types or other objects
-const providers = Object.keys(services)
+let providers = Object.keys(services)
 					.map((key) => services[key])
                     .filter((provider) => {
-                        return typeof provider === 'function';
+                        return typeof provider === 'function' && skipServices.indexOf(provider.name) === -1;
```

Comment:
żebym nie musiał dodawać serwisów i providerów dla nich oddzielnie importuje wszystkie 
```
import * as services from './services';
```
a później tylko sprawdzam czy zaimportowany kod jest funkcją (jakbym przez przypadek coś innego eksportował).

natomiast
```
&& skipServices.indexOf(provider.name) === -1;
```
filtruje wyjątki, 
np 'AuthorizedHttp' to wyjątek, który trzeba inaczej "wstrzyknąć" (poniżej)